src/main/java/com/github/igotyou/FactoryMod/utility/FactoryModGUI.java

```diff
@@ -168,6 +168,21 @@ private IClickable getSetupClick(FurnCraftChestEgg factory) {
 		return new LClickable(is, p -> showFactoryCreation(factory, true));
 	}
 
+	private IClickable getFuelClick(FurnCraftChestEgg factory, InputRecipe recipe) {
+		if (recipe == null) {
+			return null;
+		}
+		if (factory == null) {
+			return null;
+		}
+		ItemStack is = factory.getFuel();
```

Comment:
Add a `.clone()` here, otherwise you'll be modifying the ItemStack stored as fuel itself.